src/components/PaneCollapseButton.js

```diff
@@ -0,0 +1,15 @@
+const React = require("react");
+const classnames = require("classnames");
+const Svg = require("./utils/Svg");
+
+require("./PaneCollapseButton.css");
+
+function PaneCollapseButton({ position, collapsed, horizontal, handleClick }) {
+  return React.DOM.div({
+    className: classnames(`toggle-button-${position}`, {
+      collapsed, vertical: horizontal != null ? !horizontal : false }),
```

Comment:
`vertical: horizontal != null ? !horizontal : false` can be simplified :)

Perhaps:
* we pass in `vertical` to be consistent internally
* we use a default value in the params: `PaneCollapseButton({ vertical = false }) `